docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,590 @@
+## Text
+
+Therae are three Text components to display and edit text on the screen:
+
+- The [TextLabel](#textLabel) displays a short text string.

```

Comment:
- The [TextLabel](#textLabel) class provides a control that renders a short text string.

To maintain consistency:
Refer to: 
https://developer.tizen.org/dev-guide/2.4/org.tizen.ui.practices/html/native/dali/textlabel_n.htm

Reply:
I think we should check the link before the author applies this comment.

@nawab-samsung, from where did you get this link? As you can see the link is from previous site (https://developer.tizen.org...), and the version you suggested to refer is 2.4, while we are writing 5.5.
Please suggest the latest version for the reference.

Also, I think it would better if you understand the meaning of consistency,
as it doesn't always mean revive content from the discarded pages.
It neither means duplication of the same content here and there in one same page.

In the suggested page structure,
in this list we should give simple description of each section, and the link.
In each section, they should give more detailed description for each subject.

Simply writing same sentence to here and there doesn't mean consistency, but the duplication. It doesn't worth that much as it doesn't give more information while only makes the sentences long, reducing the readability. It is different with maintaining format.

**Conclusion**:
 - Please check the validity of the reference. Don't use the data out of dated.
 - Please understand the flow and the meaning that sentences are carrying of.

I think keeping this sentence as it is would be better.

Reply:
@mjincho

Reply:
@mijin85cho  thanks for your detailed information about the fact that certain parts of documentation are **discarded**. Request you to kindly let us know and guide us about the discarded parts of the documentation such that in future we won't refer to those sets of pages.

Reply:
@mijin85cho  

- Thanks for letting us know today the fact that certain parts of documentation are **discarded**. 
- Request you to kindly share with us and also guide us about the discarded parts of the documentation such that in future we won't refer to those sets of pages.  
-   We may need repository cleanup such that the pages don't show up.

Reply:
@mijin85cho  

- Thanks for letting us know today the fact that certain parts of documentation are **discarded**. 
- Request you to kindly share with us and also guide us about the discarded parts of the documentation such that in future we won't refer to those sets of pages. In addition, we may need repository cleanup such that the pages don't show up for other readers too. 

The following was updated on 30 Oct 2019:
https://developer.tizen.org/development/guides/.net-application/natural-user-interface/ui-components/textlabel

I tried to align to that style, also keeping the following comment in consideration too https://github.com/Samsung/tizen-docs/pull/1009#discussion_r377380016 . Hence the outcome is the suggested sentence which undoubtedly is much clear sentence to comprehend as it gives complete meaning for any reader to understand, thus takes away ambiguity, which can be seen in the earlier description.

Reply:
@nawab-samsung I got several notification mails containing your question, and the answers are : 
 - Please use the links from https://docs.tizen.org/...
   For now, as we moved to https://docs.tizen.org/... and hid the pages under https://developer.tizen.org/dev-guide/... ,
    the guide pages from https://developer.tizen.org/dev-guide/... can be considered as out of dated.
   (If you know the url, you can access to old pages. However we are guiding new links instead.)
 - We are noticed now that with searching "tizen text label", we can access to old pages.
   We requested to remove the pages under https://developer.tizen.org/dev-guide/ from the searching target.

Reply:
@mijin85cho  

- Thanks for letting me know that certain parts of documentation are **discarded**, however could we add a note to these discarded docs such that any end user will also get to know that some parts of docs are discarded. 
 
- I tried to align to style in the following latest doc: 
https://developer.tizen.org/development/guides/.net-application/natural-user-interface/ui-components/textlabel (updated on 30 Oct 2019)

And also kept the following comment in consideration too while providing the sentence: 
https://github.com/Samsung/tizen-docs/pull/1009#discussion_r377380016 . 

Will keep the sentence as you say. The same was already suggested by me in my review-1 before!

Reply:
@mijin85cho should we keep lines 5 & 6 as they are. I had suggested minimal changes to the developer which were already incorporated.

Reply:
> however could we add a note to these discarded docs ...

=> For pages under https://developer.tizen.org/dev-guide/2.4/.... We will do no more additional work to the past version, to focus on the latest docs. As I mentioned above, we will hide them from Google search result.

> I tried to align to style as per following latest doc: ...

=> The author is rewriting that pages on docs.tizen.org with this PR, #1009. 
(However, as I said, we don't suggest links under https://developer.tizen.org/development/guides... anymore. The updated pages will be displayed on docs.tizen.org/...)
So it is meaningless to check the previous latest style. Why I asked to check the consistency is to get the unity between pages, but not to be obsessed to each sentences. Please understand the difference.

I suggested to keep the current sentence, because it is more simple, while containing the same meaning. We are working with developer doc, so simple/easy sentence is much better than complicated/rhetorical one.
Of course the accurate sentence is important, but that doesn't mean complex sentence is recommended.

Reply:
@mijin85cho thanks for clearing my doubts.